CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
 Changes by Version
 ==================
 
-0.34.1 (unreleased)
+0.35.1 (unreleased)
 -------------------
+
 - Nothing yet.
 
-0.34.0 (2018-03-04)
+
+0.35.0 (2019-04-29)
+-------------------
+
+- Concurrency improvements to RemoteControlledSampler ([#609](https://github.com/jaegertracing/jaeger-client-java/pull/609), [@yborovikov](https://github.com/yborovikov))
+- Adding support for opentracing 0.32 ([#567](https://github.com/jaegertracing/jaeger-client-java/pull/567), [@dougEfresh](https://github.com/dougEfresh))
+- Replacing the wrong link with the correct. ([#604](https://github.com/jaegertracing/jaeger-client-java/pull/604), [@sosiska](https://github.com/sosiska))
```

Comment:
Although all contributions are welcome and appreciated, not sure this update of the pull request template should be listed in the changelog?